.tekton/lightspeed-service-pull-request.yaml

```diff
@@ -116,6 +116,11 @@ spec:
       description: Build a source image.
       name: build-source-image
       type: string
+    - default: ""
+      description: Path to a file with build arguments which will be passed to podman
```

Comment:
What is this file? Is it in our repo?

Reply:
idk, hoping that's something Konflux will take of, if it's needed at all